lib/spec/simpleofflinecache.sw-spec.js

```diff
@@ -101,4 +104,36 @@ describe('SimpleOfflineCache instance', function () {
     });
   });
 
+  if (!isChrome) {
+    it('should respect ignoreSearch when matching', function(done) {
```

Comment:
Again, you are totally right, had this just to check that chrome is not implementing it, but as you commented this is something that should be checked by the browsers own unit test.